Library/Homebrew/cleanup.rb

```diff
@@ -50,7 +50,7 @@ def self.cleanup_cache
           cleanup_path(path) { path.unlink }
           next
         end
-        if path.basename.to_s == "java_cache" && path.directory?
+        if %w[java_cache npm_cache].include?(path.basename.to_s) && path.directory?
```

Comment:
@UniqMartin may have feelings here. I think I got told off trying to do something similar to this with `include?` last time 😄. ("Told off" was tongue-in-cheek if that doesn't translate well across text!)